multiqc_ngi/multiqc_ngi.py

```diff
@@ -443,11 +443,11 @@ def general_stats_sample_meta(self):
             }
             gsheaders['lp_concentration'] = {
                 'namespace': 'NGI',
-                'title': 'Lib Conc. ({})'.format(conc_units),
-                'description': 'Library Prep: Concentration ({})'.format(conc_units),
+                'title': f'Lib Conc. ({conc_units})',
+                'description': f'Library Prep: Concentration ({conc_units})',
                 'min': 0,
                 'scale': 'YlGn',
-                'format': '{:.,0f}',
+                'format': '{:,.0f}',
```

Comment:
Was this a typo? I have no idea which one is correct.

Reply:
It was a typo. It was correct on line 442 `'format': '{:,.2f}'` but not here